test/index.js

```diff
@@ -2160,12 +2161,16 @@ See https://github.com/sanctuary-js/sanctuary-def/tree/v${version}#ValidDate for
     eq (isRegexFlags ('gm'), true);
     eq (isRegexFlags ('im'), true);
     eq (isRegexFlags ('gim'), true);
+    eq (isRegexFlags ('dgimsuy'), true);
+    eq (isRegexFlags ('dgimsvy'), parseInt (process.versions.node, 10) >= 20);
     //  String objects are not acceptable.
     eq (isRegexFlags (new String ('')), false);
     //  Flags must be alphabetically ordered.
     eq (isRegexFlags ('mg'), false);
-    //  "Sticky" flag is not acceptable.
-    eq (isRegexFlags ('y'), false);
+    //  Flags cannot contain repeated characters.
+    eq (isRegexFlags ('gg'), false);
+    //  Flags cannot contain both 'u' and 'v'.
+    eq (isRegexFlags ('uv'), false);
```

Comment:
...it might be worth adding a test for that, because these tests all pass the flags in the same order

Reply:
The existing `'mg'` test case should suffice. :)

Reply:
Oh, indeed!